pkg/slo-controller/nodeslo/extender_config.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodeslo
+
+import (
+	"fmt"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+	slov1alpha1 "github.com/koordinator-sh/koordinator/apis/slo/v1alpha1"
+)
+
+var (
+	defaultExtensionStrategy = slov1alpha1.ExtensionsMap{}
+	defaultExtensionCfg      = extension.ExtensionCfgMap{}
+)
+
+func getDefaultExtensionStrategy() *slov1alpha1.ExtensionsMap {
```

Comment:
seems that `getDefaultExtensionStrategy` and `defaultExtensionStrategy` only used in UT, which can be deleted